packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -247,6 +294,130 @@ enum ListTileControlAffinity {
 /// ```
 /// {@end-tool}
 ///
+/// If a ListTile is not what you are looking for, custom list items can be created
```

Comment:
It the way ListTile pads and positions it's elements isn't quite what you're looking for, it's easy to create ....